src/scss/pcui-storybook.scss

```diff
@@ -3,6 +3,10 @@
 @import 'pcui-common.scss';
 @import 'pcui-hidden.scss'; // this should be last
 
+.font-regular, .font-bold, .font-thin, .font-light {
+    font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
```

Comment:
If we're adding those here maybe we can include the Montserrat font as well in this bit?

Reply:
Yeah sure, i'll add those fonts in here.